### DISCUSSION TITLE
Time Series with rotating Y axis for compass values?

### DISCUSSION THREAD

**Discussion by zubenubi** [grafana/grafana#38366] [Dashboards, Panels & Visualizations]
I am using grafana currently (version 6.6.1) and am missing one *critical* feature for my data.

A lot of my data is expressed in compass degrees, which have a range of 0-359. 

When I try to visualize trends for that data and the values are consistently from the 'North' (ranging from 345 to 15), the time series strip chart ends up looking like the result of someone lying to a lie-detector ;)

I would like to request that it be possible for the Y axis to be 'rotating' - if you will - so that a trend around 'North' can be presented more in line with what a trend around 'South' looks like today.

**Comment by zubenubi** [comment]
Here is an example of what it looks like today trying to use data expressed in compass degrees as the data moves through 'North'

![image](https://user-images.githubusercontent.com/8655783/74977827-7a2db780-53f9-11ea-8441-98c966518657.png)

**Comment by zubenubi** [comment]
This is what it looks like today when the data are moving around 'SouthWest'
![image](https://user-images.githubusercontent.com/8655783/74978173-1061dd80-53fa-11ea-8c75-7b6b4d48cde5.png)

**Comment by torkelo** [comment]
Can you describe how this would work ? A special form of y scale transform?

**Comment by zubenubi** [comment]
The time series graph Y axis labels would 'wrap' so that if the data over the time period went from 345 degrees to 15 degrees, the center of the Y axis would show a label of 0 (North) and the Y axis labels would be (roughly) 45 to 335.

Currently the Y axis looks at the full range of observed and always labels from Min to Max, so data points that - in time) go from 358 to 3 to 356 then to 2 are shown at the extremes of the plot (high and low), when in reality it is just 5 or so degrees of change.

The attached image demonstrates (although in vertical form - which would also be a nice option to have in grafana) for the TWD (true wind direction) and TWS (true wind speed).
![screenshot_2018-10-24_12 59 26](https://user-images.githubusercontent.com/8655783/75040033-eb1bb080-5487-11ea-890e-1d3e9121b114.png)

**Comment by zubenubi** [comment]
I guess that compass data is 'special' like this, which may mean that it should have its own data type, similar to integer, float, string, [compass]?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
